### PR TITLE
#198 Refine the list view

### PR DESCRIPTION
**Pull request by ricardoom**
this has been hanging out for  a minute @aslagle take a look and let me know if we can merge, and then lets huddle and figure out the next steps...

### CODE REVIEW

**Comment by ricardoom** [comment]
@aslagle, maybe the master branch isnt the right one to merge to? Let me know which one you prefer and i'll update the PR

**Comment by aslagle** [comment]
@ricardoom master is correct

**Comment by aslagle** [comment]
The search box shouldn't overflow its parent when it wraps:

screen shot 2017-10-05 at 1 38 05 pm

**Comment by aslagle** [comment]
The last line of the summary is getting cut off in list view:

screen shot 2017-10-05 at 1 40 24 pm

**Comment by aslagle** [comment]
The barberpole stripes got a lot thinner now that the popup is shorter - can we up the width so it's closer to what it was before?

**Comment by aslagle** [comment]
I feel like the position of the borrow buttons in the list is a little weird on some books... it would be nice if the borrow was always in the same place but I don't know where it could go.

screen shot 2017-10-05 at 1 44 55 pm

**Comment by aslagle** [comment]
Title should probably not pass the bar on the right?

screen shot 2017-10-05 at 1 50 18 pm

**Comment by aslagle** [comment]
Here's a book with no metadata:

screen shot 2017-10-05 at 1 52 31 pm

File: packages/opds-web-client/src/stylesheets/_variables.scss [hunk 1]
```diff
@@ -34,7 +36,7 @@ $footerHeight: 50px !default;
 // book stuff
 $bookHeight: 200px !default;
 $bookWidth: 150px !default;
-$stripeWidth: 200px !default;
+$stripeWidth: 50px !default;
```
Comment (aslagle) [inline packages/opds-web-client/src/stylesheets/_variables.scss]: Oh you actually made the stripes smaller... I liked the wider ones :(

File: packages/opds-web-client/src/stylesheets/book.scss [hunk 2]
```diff
@@ -17,24 +18,53 @@
...
       border-color: $linkhovercolor;
-      color: $linkhovercolor;
+      color: $invertedlinkcolor;
+    }
+
+    &::after {
+      content: " ";
+      width: 10px;
+      height: 10px;
+      background: red;
```
Comment (aslagle) [inline packages/opds-web-client/src/stylesheets/book.scss]: what's this?

File: packages/opds-web-client/src/stylesheets/book.scss [hunk 3]
```diff
@@ -164,8 +222,8 @@
...
-}
+} // /.list-view
 
-.list-view li:nth-child(odd) .book .expanded-info .summary a {
-  background: linear-gradient(to right, transparent, $pagecolorlight 40%, $pagecolorlight);
-}
\ No newline at end of file
+// .list-view li:nth-child(odd) .book .expanded-info .summary a {
+//   background: linear-gradient(to right, transparent, $pagecolorlight 40%, $pagecolorlight);
+// }
```
Comment (aslagle) [inline packages/opds-web-client/src/stylesheets/book.scss]: I would take out all the commented out stuff, we don't need this anymore.

File: packages/opds-web-client/src/stylesheets/custom-themes/_theme-demo.scss [hunk 4]
```diff
@@ -0,0 +1,6 @@
+//
+// Demo theme colors
+//
+
+$blue: #3a7dfd;
+$darkblue: #234c99;
```
Comment (aslagle) [inline packages/opds-web-client/src/stylesheets/custom-themes/_theme-demo.scss]: This isn't used, right? It won't do anything unless you override some of the other vars..

**Comment by aslagle** [comment]
I left you some comments @ricardoom but I think it looks great overall. I'm going to test with the other themes too and make sure nothing breaks.

**Comment by aslagle** [comment]
The other themes look good, except for title and author ending up different colors in the book cover hover:

screen shot 2017-10-05 at 2 18 37 pm

File: packages/opds-web-client/src/stylesheets/root.scss [hunk 5]
```diff
@@ -139,24 +164,29 @@ a {
...
 // override bootstrap
-.navbar-default, .navbar-collapse {
+.navbar-default,
+.navbar-collapse {
   background-color: $dark;
   border: none;
-  color: white;
+  border-radius: 0;
+  color: $light;
+  height: 4.25rem;
```
Comment (aslagle) [inline packages/opds-web-client/src/stylesheets/root.scss]: This height is causing some problems with the header dropdown in the admin interface when it's collapsed on small screens.

**Comment by EdwinGuzman** [comment]
What's going on with this PR?

**Comment by aslagle** [comment]
Oh yeah, I forgot about this pr. @ricardoom never had time to finish it :(

**Comment by ricardoom** [comment]
closing this down...

**Pull request closed without merging**